package.json

```diff
@@ -12,20 +12,23 @@
     "@actions/core": "1.6.0",
     "@actions/github": "5.0.0",
     "@commitlint/config-conventional": "16.0.0",
-    "@commitlint/config-lerna-scopes": "16.0.0",
     "@commitlint/lint": "16.0.0",
-    "@coveord/platform-client": "28.4.0",
+    "@coveo/semantic-monorepo-tools": "file:packages/releaseLib",
+    "@coveord/platform-client": "28.5.0",
+    "@nrwl/cli": "latest",
+    "@nrwl/tao": "latest",
```

Comment:
What does this package actually do?

Reply:
good catch, it's a dep of the nrwl-cli (i.e. nx), but we don't need it in direct dep afaik